portability-transfer/src/main/java/org/dataportabilityproject/transfer/PortabilityInMemoryDataCopier.java

```diff
@@ -42,6 +43,9 @@
   private static final AtomicInteger COPY_ITERATION_COUNTER = new AtomicInteger();
   private static final Logger logger = LoggerFactory.getLogger(PortabilityInMemoryDataCopier.class);
 
+  private static final List<String> fatalRegexes = ImmutableList.of("*fatal*");
```

Comment:
can fatalRegexes (the var name) be more descriptive? as a member of InMemoryDataCopier, I don't think there is much context

Reply:
Is fatalErrorRegexes sufficiently descriptive?